allenact/base_abstractions/sensor.py

```diff
@@ -393,27 +393,3 @@ def query_expert(
         return task.query_expert(
             **self.expert_args, expert_sensor_group_name=expert_sensor_group_name
         )
-
-
-class VisionSensor:
```

Comment:
I know I was the one originally arguing for forwarding users in this way but it ended up causing more problems than it was worth (Kiana/I independently ran into problems where doing this type of forwarding resulted in weird issues, for me this was when some old code was doing something like `isinstance(obj, RGBSensor)` and then failing subtly because `RGBSensor` was one of these stubs rather than the true `RGBSensor`.